index.md

```diff
@@ -36,7 +36,7 @@ Additional context and background can be found in the [foreword](foreword.md).
 ## Community calls
 
 We usually have a community call on the first Thursday of the month at 15:00 (CET/CEST).
-[The agenda](https://write.publiccode.net/pads/Community-Call-Standard-for-Public-Code) is updated roughly a week before the call.
+The agenda is coordinated on our [discussion board](https://github.com/standard-for-public-code/standard-for-public-code/discussions) roughly a week before the call.
```

Comment:
```suggestion
The agenda is coordinated on our [discussion board](https://github.com/standard-for-public-code/standard-for-public-code/discussions/categories/community-calls) roughly a week before the call.
```
Making the link directly to the community calls category should ensure that even if a lot of discussions are going on about other things, it will be easy to spot the agenda.